airbyte-integrations/connectors/source-outreach/source_outreach/manifest.yaml

```diff
@@ -0,0 +1,5755 @@
+            - integer
+      mailings:
+        type:
+          - "null"
+          - array
+        items:
+          type:
+            - "null"
+            - integer
+      opportunity:
+        type:
+          - "null"
+          - array
+        items:
+          type:
+            - "null"
+            - integer
+      prospect:
+        type:
+          - "null"
+          - array
+        items:
+          type:
+            - "null"
+            - integer
+      sequence:
+        type:
+          - "null"
+          - array
+        items:
+          type:
+            - "null"
+            - integer
+      sequenceStep:
+        type:
+          - "null"
+          - array
+        items:
+          type:
+            - "null"
+            - integer
+      tasks:
+        type:
+          - "null"
+          - array
+        items:
+          type:
+            - "null"
+            - integer
+
+  sequence_steps_schema:
+    $schema: http://json-schema.org/draft-07/schema#
+    type: object
+    additionalProperties: true
+    properties:
+      id:
+        type: integer
+      bounceCount:
+        type:
+          - "null"
+          - integer
+      clickCount:
+        type:
+          - "null"
+          - integer
+      createdAt:
+        type:
+          - "null"
+          - string
+        format: date-time
+      date:
+        type:
+          - "null"
+          - string
+        format: date
+      deliverCount:
+        type:
+          - "null"
+          - integer
+      displayName:
+        type:
+          - "null"
+          - string
+      failureCount:
+        type:
+          - "null"
+          - integer
+      interval:
+        type:
+          - "null"
+          - integer
+      negativeReplyCount:
+        type:
+          - "null"
+          - integer
+      neutralReplyCount:
+        type:
+          - "null"
+          - integer
+      openCount:
+        type:
+          - "null"
+          - integer
+      optOutCount:
+        type:
+          - "null"
+          - integer
+      order:
+        type:
+          - "null"
+          - integer
+      positiveReplyCount:
+        type:
+          - "null"
+          - integer
+      replyCount:
+        type:
+          - "null"
+          - integer
+      scheduleCount:
+        type:
+          - "null"
+          - integer
+      stepType:
+        type:
+          - "null"
+          - string
+      taskAutoDisplay:
+        type:
+          - "null"
+          - string
+      updatedAt:
+        type:
+          - "null"
+          - string
+        format: date-time
+      callPurpose:
+        type:
+          - "null"
+          - array
+        items:
+          type: integer
+      calls:
+        type:
+          - "null"
+          - array
+        items:
+          type: integer
+      creator:
+        type:
+          - "null"
+          - array
+        items:
+          type:
+            - string
+            - integer
```

Comment:
Leave one of data type and use transformer to update field data to schema filed type

Reply:
There is a mismatch, there are not strings In that, only integers, still a breaking change, but no need to transformer I think, Corrected!